nomos-services/api/src/http/da.rs

```diff
@@ -269,3 +280,89 @@ where
 
     receiver.await?
 }
+
+pub async fn block_peer<B>(handle: &OverwatchHandle, peer_id: PeerId) -> Result<bool, DynError>
+where
+    B: NetworkBackend + 'static + Send,
+    B::Message: PeerMessagesBuilder,
+{
+    let relay = handle.relay::<NetworkService<B>>().connect().await?;
+    let (sender, receiver) = oneshot::channel();
+    let message = B::Message::create_block_message(peer_id, sender);
+    relay
+        .send(DaNetworkMsg::Process(message))
+        .await
+        .map_err(|(e, _)| e)?;
+    Ok(receiver.await?)
```

Comment:
Timeout (internal server error) should be added here imho, but I left it for a different PR as I would like to add it to other http calls as well, if it makes sense.

Reply:
Let's open an issue to track it. We can add it to the next iteration. Thanks

Reply:
https://github.com/logos-co/nomos/issues/1137